plugins/native-script-example/example/primes.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+curl -s -XDELETE "http://localhost:9200/test"
```

Comment:
Instead of having these shell tests, I think we should show how to write unit tests?